sites/partners/src/pages/listings/[id]/lottery.tsx

```diff
@@ -378,6 +455,53 @@ const Lottery = (props: { listing: Listing }) => {
               </Button>
             </Dialog.Footer>
           </Dialog>
+          <Dialog
+            isOpen={!!publishModal}
+            ariaLabelledBy="publish-lottery-modal-header"
+            ariaDescribedBy="publish-lottery-modal-content"
+            onClose={() => setPublishModal(false)}
+          >
+            <Dialog.Header id="publish-lottery-modal-header">
+              {t("applications.addConfirmModalHeader")}
+            </Dialog.Header>
+            <Dialog.Content id="publish-lottery-modal-content">
+              <p>{t("listings.lottery.publishContent")}</p>
+              <p>{t("applications.addConfirmModalAddApplicationPostLotteryAreYouSure")}</p>
+            </Dialog.Content>
+            <Dialog.Footer>
+              <Button
+                variant="primary"
+                onClick={async () => {
+                  setLoading(true)
+                  try {
+                    await listingsService.lotteryStatus({
+                      body: {
+                        listingId: listing.id,
+                        lotteryStatus: LotteryStatusEnum.publishedToPublic,
+                      },
+                    })
+                    location.reload()
```

Comment:
Is the reload so that a new call to the backend is triggered?

Reply:
We just need to refresh the page